static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -267,6 +324,30 @@ define([
                     } else {
                         resolve();
                     }
+
+                    if (abVideoAutoplay) {
+                        // Annoyingly we pass the `parentNode` as the video is absolutely positioned.
+                        var parentNode = player.el().parentNode;
+                        var firstAutoplay = true;
+                        var elementInView = ElementViewable(parentNode, parentNode.clientHeight * (3 / 4), function () {
+                            if (firstAutoplay) {
+                                player.volume(0);
+                                firstAutoplay = false;
+                            }
+                            player.play();
+                        });
+                        elementInView.on('viewenter', function autoplayInView() {
+                            if (firstAutoplay) {
```

Comment:
Yea, if I were doing this properly I would be using our [mini-redux](https://github.com/guardian/frontend/blob/article-video-autoplay/static/src/javascripts/projects/common/utils/create-store.js)